framework/SwiftPlot/Pair.swift

```diff
@@ -0,0 +1,12 @@
+// struct defining a Pair
+public struct Pair<T,U> {
+    public let x: T
+    public let y: U
+
+    public init(_ x: T, _ y: U){
+        self.x = x
+        self.y = y
+    }
+}
+
+let zeroPair = Pair<FloatConvertible,FloatConvertible>(0.0, 0.0)
```

Comment:
Can this be done as a static let, such as the previous Point.zero? I believe you should be able to use that with a Pair<Float, Float> type in the static let.

Reply:
I tried this. But am getting this error:
`static stored properties not supported in generic types`

Reply:
Hmm, guess that won't work then. Think I know why that's happening. Oh, well.